types/is-negated-glob/is-negated-glob-tests.ts

```diff
@@ -0,0 +1,6 @@
+import * as isNegatedGlob from 'is-negated-glob';
```

Comment:
Since `isNegatedGlob` is function, it should be `import isNegatedGlob = require(...)` here